src/components/CustomerDetails/CollectCustomerDetailsScreen.tsx

```diff
@@ -80,7 +80,7 @@ const styles = StyleSheet.create({
     marginBottom: size(3),
   },
   statsButton: {
-    marginTop: size(4),
+    marginTop: size(2),
```

Comment:
I feel that with `size(2)`, the lower side of the screen feels empty. Should it be kept at `size(4)`? What do you think?

Reply:
Have checked with Eida regarding this and have updated the sizes!